_test/markdown-linter-style.rb

```diff
@@ -0,0 +1,11 @@
+all
+exclude_rule 'MD001'
+rule 'MD009', :br_spaces => 2
+exclude_rule 'MD013'
+exclude_rule 'MD022'
+exclude_rule 'MD026'
+rule 'MD029', :style => :ordered
+exclude_rule 'MD032'
+exclude_rule 'MD033'
+exclude_rule 'MD034'
+exclude_rule 'MD036'
```

Comment:
Comments added in https://github.com/shredtechular/m-lab.github.io/commit/5ac4e73b2716cce1f79a34e5219c6dbac4d91e6c